build-utils/integration-docs-fetch-plugin.ts

```diff
@@ -2,12 +2,16 @@
 /* eslint import/no-nodejs-modules:0 */
 
 import fs from 'fs';
+import http from 'http';
 import https from 'https';
 import path from 'path';
+import url from 'url';
 
 import webpack from 'webpack';
 
-const PLATFORMS_URL = 'https://docs.sentry.io/_platforms/_index.json';
+const INTEGRATIONS_DOC_URL =
+  process.env.INTEGRATION_DOCS_URL || 'https://docs.sentry.io/_platforms/';
+const PLATFORMS_URL = INTEGRATIONS_DOC_URL + '_index.json';
```

Comment:
allows you to stay in sync with a local version of `sentry-docs` as long as we set `INTEGRATIONS_DOC_URL`

```bash
export INTEGRATION_DOCS_URL=http://127.0.0.1:8080/_platforms/
#... start dev server
```